lift/Readme.md

```diff
@@ -245,9 +253,11 @@ migrations/
 
 A migration folder contains 3 files:
 
-- **steps.json:** contains a JSON list of steps to run against the database
-- **schema.prisma:** contains a snapshot of your `schema.prisma` file at a specific point in time.
+- **steps.json:** contains a JSON list of steps to run against the database. Steps contains only the up steps, the down steps are calculated on the fly.
```

Comment:
Is that technically correct @mavilein? Or is this more general and both steps are generated?

Reply:
It's a little bit more involved. The `steps.json` only represents the changes that should be applied to the current datamodel to deduct the desired datamodel. Based on the current and desired datamodel we generate the actual database steps that will be performed for up and down.
To put it into equations:
1. `desired_datamodel = current_datamodel + steps.json`
2. `actual_database_steps = generate(current_datamodel, desired_datamodel)` <- this function is connector specific

Does that explanation help?

Reply:
It does and gives me details, thanks! And it also kind of confirms that "up steps" is valid enough here in what @matthewmueller wrote - it contains things to go from current to desired, which I think is fine to describe as "up steps", right?

Reply:
Yes i think it's accurate enough.